bitnami/vault/templates/server/statefulset.yaml

```
@@ -274,5 +274,8 @@ spec:
         {{- if .Values.server.persistence.selector }}
         selector: {{- include "common.tplvalues.render" (dict "value" .Values.server.persistence.selector "context" $) | nindent 10 }}
         {{- end }}
+        {{- if .Values.server.persistence.extraVolumeClaimTemplates }}
+        {{- include "common.tplvalues.render" (dict "value" .Values.server.persistence.extraVolumeClaimTemplates "context" $) | nindent 4 }}
+        {{- end }}
```

Sorry @Archanashetty7 The indentation in my first comment was wrong, my failure. It should be like this:

```suggestion
    {{- if .Values.server.persistence.extraVolumeClaimTemplates }}
    {{- include "common.tplvalues.render" (dict "value" .Values.server.persistence.extraVolumeClaimTemplates "context" $) | nindent 4 }}
    {{- end }}
```